_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -170346,20 +170412,20 @@ aZL
 bbu
 aZL
 aKV
-aad
-ajr
-ajr
-ajr
-aad
-ajr
-ajr
-ajr
-ajr
-ajr
-ajr
-aad
+qYo
+efQ
+efQ
+efQ
+qYo
+efQ
```

Comment:
![image](https://user-images.githubusercontent.com/6209658/129812670-fca6eb4c-696c-418e-81c1-a1651e44e617.png)
all these too.

Reply:
Thanks for the catch, I'll get right on it 👍